stdlib/REPL/src/REPL.jl

```diff
@@ -1239,4 +1242,15 @@ function start_repl_server(port::Int)
     end
 end
 
+print_exit_hint(repl::BasicREPL, line) = print_exit_hint(repl.terminal, line)
+print_exit_hint(repl::LineEditREPL, line) = print_exit_hint(repl.t, line)
+print_exit_hint(repl::StreamREPL, line) = print_exit_hint(repl.stream, line)
+function print_exit_hint(io::IO, line)
+    if lowercase(strip(line)) == "exit"
+        println(io)
+        println(io, "To exit Julia, type exit() and press enter")
```

Comment:
Maybe mention better ways to exit?
```suggestion
        println(io, "To exit Julia use Ctrl-D, or type exit() and press enter")
```

Reply:
I don't know if this works on Windows though.

Reply:
We can do a Windows check, and only mention this if it isn't Windows.

Reply:
Take a look now. We only mention `Ctrl-D` if it is not Windows.